tests/utils/index.js

```diff
@@ -130,7 +126,7 @@ const setupUserDoc = (userName = constants.USERNAME, userDoc = userSettings.buil
 
 // First Object is passed to http.request, second is for specific options / flags
 // for this wrapper
-const request = (options, { debug } = {}) => {
+const request = (options, { debug } = {}) => { //NOSONAR
```

Comment:
Are all these sonar skips about the same warning? What was Sonar's complain?

Reply:
Complexity ... it would have required a complete rewrite of all these functions, which i think is out of scope for this PR.

Reply:
Maybe open a ticket to reduce complexity on those functions later

Reply:
done https://github.com/medic/cht-core/issues/8769